pkg/grpc/api.go

```diff
@@ -163,6 +164,14 @@ func (a *api) CallLocal(ctx context.Context, in *internalv1pb.InternalInvokeRequ
 	return resp.Proto(), err
 }
 
+func normalizeOperation(operation string) string {
+	s, err := purell.NormalizeURLString(operation, purell.FlagsUsuallySafeGreedy|purell.FlagRemoveDuplicateSlashes)
+	if err != nil {
+		panic(err)
```

Comment:
i think this should be an error returned. 
In line 188, surround by if clause and return err message to caller.

Reply:
oops, yes. that should be a return